test/lib/helpers.js

```diff
@@ -0,0 +1,37 @@
+import { isExisty } from '../../lib/helpers';
+
+describe('isExisty', () => {
+  describe('existy things', () => {
+    it('returns `true` for Integers', () => {
+      isExisty(0).should.be.true();
+      isExisty(100).should.be.true();
+    });
+
+    it('returns `true` for Strings', () => {
+      isExisty('0').should.be.true();
+      isExisty('Foobar').should.be.true();
+    });
+
+    it('returns `true` for `NaN`', () => {
+      isExisty(NaN).should.be.true();
+    });
+
+    it('returns `true` for non-empty Objects', () => {
+      isExisty({ foo: 'bar' }).should.be.true();
+    });
+  });
+
+  describe('not existy things', () => {
+    it('returns `false` for empty Objects', () => {
+      isExisty({}).should.be.false();
```

Comment:
I realize empty objects strictly *exist*—perhaps this predicate needs a rename? Ideas?